schema/SCHEMA_QUDT-v2.1.ttl

```diff
@@ -1,4 +1,5 @@
 # baseURI: http://qudt.org/2.1/schema/qudt
+# imports: http://qudt.org/2.1/schema/extensions/imports
```

Comment:
propose http://qudt.org/2.1/schema/qudt/extensions

rationale: 

1. same as qudt schema base uri up to /extensions
2. extensions can be more than graph imports - if people choose to do that

Reply:
Because extensions can be more than just imports is why I called this one "imports", allowing the existence of other things in the directory as well.

Reply:
Agreed that this imports file/URI will only contain import statements.
An example import would be the qudt skos extension file.